setup.sh

```diff
@@ -9,6 +9,7 @@ sudo -n dnf install -y pipenv
 cd /vagrant
 # Install dependencies with Pipenv
 pipenv sync --dev
+pipenv install pillow
```

Comment:
Is possible to keep these dependencies in the Pipfile?

Reply:
> Is possible to keep these dependencies in the Pipfile?

Hi, I would like to close this PR if possible, enabling use of images is part of #34